addons/mail/models/fetchmail.py

```diff
@@ -250,8 +253,11 @@ def fetch_mail(self):
                         if num_messages < MAX_POP_MESSAGES or failed_in_loop == num:
                             break
                         pop_server.quit()
-                except Exception:
-                    _logger.info("General failure when trying to fetch mail from %s server %s.", server.server_type, server.name, exc_info=True)
+                except Exception as e:
+                    if raise_exception:
+                        raise ValidationError(_("Error when fetching mail, here is the debugging information: %s") % (str(e)))
```

Comment:
`%s` will already call the `__str__`

not 100% sure about the `from e` (to double check)

```suggestion
                        raise ValidationError(_("Error when fetching mail, here is the debugging information: %s", e)) from e

Reply:
Done sir